### ISSUE TITLE
How to enable mpv player?

### ISSUE THREAD

**Issue by LeonCheung** [SubtitleEdit/subtitleedit#2242]
I have mpv player installed on my win8.1 64bit PC, but the mpv button is still gray, don't know how to use it, any hints please?
![mpv](https://cloud.githubusercontent.com/assets/494788/23270660/8a77db88-fa2f-11e6-8f80-8466ee5d350d.png)

**Comment by niksedk** [comment]
OK, I guess the UI could be improved, but try pressing the "..." button and then "Download".

**Comment by niksedk** [comment]
SE don't use the "mpv" player, but libmpv... (mpv-1.dll).
Is latest beta any easier to understand? https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.1/SubtitleEditBeta.zip

Otherwise I guess the help file needs updating ;)

**Comment by LeonCheung** [comment]
Yeah, much better. :)  I just failed to download anything by clicking the "..." button yesterday, so.
But it works now, and easier to understand for fure. Thank you!

**Comment by niksedk** [comment]
If think "opengl" is preferred, but "direct3d" is the most compatible...
You can read about the video output types here: https://mpv.io/manual/stable/#video-output-drivers

**Comment by LeonCheung** [comment]
Thanks, got it!
